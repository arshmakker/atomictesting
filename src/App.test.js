import React from 'react';
import renderer from 'react-test-renderer'
import App from './App';

test('Renders Header component', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot()
});
