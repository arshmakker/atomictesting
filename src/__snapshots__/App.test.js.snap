// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header component 1`] = `
<div
  className="App"
>
  <div
    style={
      Object {
        "background": "lightblue",
        "display": "flex",
        "justifyContent": "space-around",
        "margin": "20px",
      }
    }
  >
    <div>
      Ma Web
    </div>
    <img
      alt="unapologetic logo"
      className=""
      src="https://img.icons8.com/ios/50/000000/pick-up.png"
    />
    <div
      className=""
    >
      <input
        id="searchText"
        placeholder="Type here to search"
        type="text"
      />
      <button
        className=""
        onClick={[Function]}
      >
        Search
      </button>
    </div>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-around",
        "margin": "20px",
      }
    }
  >
    <div
      style={
        Object {
          "background": "lightgrey",
          "padding": "20px",
        }
      }
    >
      <ul>
        <li>
          1st Option
        </li>
        <li>
          2nd Option
        </li>
        <li>
          3rd Option
        </li>
      </ul>
    </div>
    <div
      style={
        Object {
          "background": "lightgreen",
          "padding": "20px",
        }
      }
    >
      <ul />
    </div>
  </div>
</div>
`;
